roles/openshift_logging/tasks/label_node.yaml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
@jcantrill the openshift_manage_node task in openshift-ansible can be used for managing node labels (and already is used for that). It is also possible to configure the node config in a way that will automatically add node labels on registration.

The worry that I would have is that someone would be using openshift-ansible (or the Ops tooling) and that would potentially clobber the node labeling you are doing here.

In openshift-ansible, the way we allow for setting node labels is to use the openshift_node_labels variable that can be defined for each host. @abutcher thoughts?